tests/test_C1_verification.py

```diff
@@ -586,6 +590,152 @@ def test_check_for_label_duplicates_passes():
     ), f"There is no duplicate value for label, but still an error is raised."
 
 
+def test_check_energy_system_can_fulfill_max_demand_sufficient_capacities(caplog):
+    energy_vector = "Electricity"
+    time_series = pd.Series([0, 10, 100, 10, 0])
+    dict_values = {
+        ENERGY_BUSSES: {
+            "Elec": {
+                ENERGY_VECTOR: energy_vector
```

Comment:
>the tests arent running through though because for some reason it isnt recognising 
>'Asset_list'
>for bus in dict_values[ENERGY_BUSSES]:
> for item in dict_values[ENERGY_BUSSES][bus][ASSET_DICT]:
>E KeyError: 'Asset_list'

You did not define the `ASSET_LIST` in `ENERGY_BUSSES`/`Elec` here, so it does not find it. It needs to be 

```
    ENERGY_BUSSES: {
            "Elec": {
                ENERGY_VECTOR: energy_vector,
                ASSET_DICT: {
                         "Demand": "Demand Label", 
                         "Inverter": "Inverter Label"}}}
```
You are reusing your asset name or "IDs" here, ie. `[energyConsumption][asset]` as well as the assets label ` [energyConsumption][asset][LABEL]`. If you want to make sure you dont have typos you can also define your asset names above (`asset_name = "string"`) and define the labels always as `asset_name + LABEL`